hera_sim/interpolators.py

```diff
@@ -68,6 +88,12 @@ def __init__(self, datafile, pol='xx', **interp_kwargs):
     def __call__(self, lsts, freqs):
         return self._interpolator(lsts, freqs)
 
+    # XXX is there a cleaner way of doing this? (I want to be able to find out
```

Comment:
I'm sure there is. What is the purpose of this? It seems like it should happen the other way round: the parameter that accepts it should check whether what is passed is indeed an instance of `Tsky`.